src/commands/sign-requests/create.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const { flags } = require('@oclif/command');
+
+class SignRequestsCreateCommand extends BoxCommand {
+	async run() {
+		const { flags, args } = this.parse(SignRequestsCreateCommand);
+
+		const signRequest = await this.client.signRequests.create({
```

Comment:
We don't map any other sign request fields (email_message, prefill_tags etc.)? You think that they will not be used in the CLI?